raiden/tests/integration/rpc/assumptions/test_rpc_gas_price_assumptions.py

```diff
@@ -155,7 +155,11 @@ def test_local_transaction_with_zero_gasprice_is_mined(deploy_client: JSONRPCCli
 
 @pytest.mark.parametrize("blockchain_number_of_nodes", [2])
 def test_remote_transaction_with_zero_gasprice_is_not_mined(
-    web3: Web3, deploy_key: PrivateKey, blockchain_rpc_ports: List[Port], blockchain_type: str
+    web3: Web3,
+    deploy_key: PrivateKey,
+    blockchain_rpc_ports: List[Port],
+    blockchain_type: str,
+    skip_if_not_parity,
```

Comment:
Can you also make an issue and link it here with a comment?

Reply:
Done

Reply:
I meant a comment in the code. Generally if I see a skip in the code I want to see an issue next to it.